### ISSUE TITLE
 error drawContour

### ISSUE THREAD

**Issue by andung** [PROpraveee/OpenCV-Projects#1]
error: (-215:Assertion failed) reader.ptr != NULL in function 'cvDrawContours'
line 39

**Comment by phernady** [comment]
> error: (-215:Assertion failed) reader.ptr != NULL in function 'cvDrawContours'
> line 39

I have the same issue. 
Somehow the cv2 does not find the license plate as a contour with the original examle image.
Even removing the limit at the sorting, only a small part in the headlight is recognized.
Version: opencv-python==4.4.0.40

Maybe some tuning needed at the filtering or edging?

**Comment by whodanyalahmed** [comment]
same error 
@GeekyPRAVEE please solve this issue

**Comment by arya312** [comment]
yes im facing the same problem. the draw contour is not detecting any contour

**Comment by wild-civil** [comment]
> error: (-215:Assertion failed) reader.ptr != NULL in function 'cvDrawContours' line 39

I change the picture, and it worked.
